bridges/snowbridge/primitives/inbound-queue/src/v2/traits.rs

```diff
@@ -21,3 +22,59 @@ pub enum ConvertMessageError {
 	/// Invalid network specified (not from Ethereum)
 	InvalidNetwork,
 }
+
+/// Reason why a message processor failed.
+#[derive(Clone, RuntimeDebug, PartialEq)]
+pub enum MessageProcessorError {
+	/// Message processing failed.
+	ProcessMessageError(DispatchError),
+	/// Message conversion failed.
+	ConvertMessageError(ConvertMessageError),
+	/// Message sending failed.
+	SendMessageError(SendError),
```

Comment:
small nit: Can reduce the amount of redundant `Error` words:

```suggestion
	ProcessMessage(DispatchError),
	/// Message conversion failed.
	ConvertMessage(ConvertMessageError),
	/// Message sending failed.
	SendMessage(SendError),
```

Reply:
Sure! Done in 7674a2b3f587fa56b56cbac343cc2582bc934062.